### PR TITLE
EPS-612: Improved codebase

### PR DESCRIPTION
**Pull request by vijetaR**
Improved codebase

### CODE REVIEW

**Review by akshayurankar48 — APPROVED** [review]

File: bb-bootstrap-alerts-module/includes/frontend.php [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
 	<?php if($settings->bbn_dropdown_link=='yes'): ?>
-		<a href="<?php echo ($nav_link=$settings->bbn_navigation_link != '')? $nav_link=$settings->bbn_navigation_link : '#' ;?>" target="<?php echo $nav_target=$settings->bbn_navigation_target;?>" >
+		<a href="<?php echo esc_url($nav_link=$settings->bbn_navigation_link != '')? $nav_link=$settings->bbn_navigation_link : '#' ;?>" target="<?php echo esc_attr( $nav_target=$settings->bbn_navigation_target );?>" >
 	<?php endif; ?>
 			<!-- Alert class -->
-			<div class="alert <?php echo $settings->bbn_dropdown_field; ?> fade in" >
+			<div class="alert <?php echo esc_attr( $settings->bbn_dropdown_field ); ?> fade in" >
 				<!-- Close Button -->
-				<?php if( $settings->bbn_closable == 'yes' ): ?>
+				<?php if( esc_attr( $settings->bbn_closable == 'yes' ) ): ?>
```
Comment (sushmak02) [inline bb-bootstrap-alerts-module/includes/frontend.php:19]: I think  $settings->bbn_closable only  should go through esc_attr()

why whole if condition is added inside it?

File: bb-bootstrap-alerts-module/includes/frontend.php [hunk 2]
```diff
@@ -25,16 +25,16 @@
 	    			<div class="alert-padding">
 
 	    			<!-- Before text icon -->
-	    			<?php if( $settings->bbn_icon_align == 'before' ): ?>
-						<i class="<?php echo $settings->bbn_icon_field; ?>"></i>
+	    			<?php if( esc_attr( $settings->bbn_icon_align == 'before' ) ): ?>
```
Comment (sushmak02) [inline bb-bootstrap-alerts-module/includes/frontend.php:28]: same suggestion

File: bb-bootstrap-alerts-module/includes/frontend.php [hunk 3]
```diff
@@ -25,16 +25,16 @@
...
+						<i class="<?php echo esc_attr( $settings->bbn_icon_field ); ?>"></i>
 					<?php endif; ?>
 
 					<!-- Alert Contents -->
 	    			<?php echo $settings->bbn_textarea_field; ?>
 
 	    			<!-- After text icon -->
-	    			<?php if( $settings->bbn_icon_align == 'after' ): ?>
-	    				<i class="<?php echo $settings->bbn_icon_field; ?>"></i>
+	    			<?php if( esc_attr( $settings->bbn_icon_align == 'after' ) ): ?>
```
Comment (sushmak02) [inline bb-bootstrap-alerts-module/includes/frontend.php:36]: same

File: bb-bootstrap-alerts-module/includes/frontend.php (hunk 1 above)
Comment (vijetaR) [reply bb-bootstrap-alerts-module/includes/frontend.php:19]: Yes, correct. Will update
sanitizing `esc_attr` only on `$settings->bbn_closable` to make sure its value is safe for HTML attributes and ensure security.

File: bb-bootstrap-alerts-module/includes/frontend.php (hunk 2 above)
Comment (vijetaR) [reply bb-bootstrap-alerts-module/includes/frontend.php]: Updated

File: bb-bootstrap-alerts-module/includes/frontend.php (hunk 3 above)
Comment (vijetaR) [reply bb-bootstrap-alerts-module/includes/frontend.php]: Updated

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**